core-tests/src/test/java/org/mule/runtime/core/internal/connection/PoolingConnectionManagementStrategyTestCase.java

```diff
@@ -96,11 +98,14 @@ public void poolingListenerFailsOnBorrow() throws Exception {
       assertThat(e.getCause(), is(sameInstance(exception)));
       verify(connectionProvider).disconnect(any(Lifecycle.class));
     }
-
   }
 
   @Test
   public void connectionsDependenciesInjected() throws Exception {
+    initStrategy();
```

Comment:
why adding this here? This already happens on the before()

Reply:
removed it from the before() because some tests need to configure additional things before initializing the strategy

Reply:
if only a couple of tests need a different setting then leave it on before and let that handful of test redefine only when needed

Reply:
It is more than just a couple of tests. It is shorter this way and it also avoids reseting and re-initializing objects in the 6 new tests. Plus I think is is better for the @before method to just contain code that is common to ALL tests.